.gitmodules

```diff
@@ -19,3 +19,6 @@
 [submodule "3rdparty/OpenCL-Headers"]
 	path = 3rdparty/OpenCL-Headers
 	url = https://github.com/KhronosGroup/OpenCL-Headers.git
+[submodule "3rdparty/cnpy"]
+	path = 3rdparty/cnpy
+	url = https://github.com/rogersce/cnpy.git
```

Comment:
I was generally less aggressive when introducing new dependency. It doesn't mean I don't like it in this case, but do you think we could use TVM's native NDArray saving methods instead?

Reply:
I considered it initially to reuse what is available in TVM. But, this require additional effort from the developer/user to parse NDArray dump. Or we need to give a python utility that converts it to a numpy dump.

Reply:
Thanks for the context! If so, I do not object the idea as long as it's not part of default build

Reply:
Personally I still think having an util to be able to create such a dump can be helpful. Mainly because the numpy format can be complicated and there will be questions of supporting other formats, such as protobuf. So converting to a standard one that we commonly use can be better.